src/renderer/views/singleplayer/scenarios.vue

```diff
@@ -74,7 +74,8 @@ import Markdown from "@renderer/components/misc/Markdown.vue";
 import DownloadContentButton from "@renderer/components/controls/DownloadContentButton.vue";
 import { gameStore } from "@renderer/store/game.store";
 
-const loadedScenarios = await window.game.getScenarios(gameStore.selectedGameVersion?.gameVersion);
+const gameVersion = gameStore?.selectedGameVersion?.gameVersion;
```

Comment:
Small tweaks to eliminate an undefined argument which the other side didn't handle.